sw/airborne/modules/lidar/maps/ucm_football.h

```diff
@@ -0,0 +1,55 @@
+// modules/lidar/maps/ucm_football.h
+
+// TODO: Maybe try to use a code to automatically generate this file
+// Its not perfect, but at least its a separated file
```

Comment:
The airframe file allows you to define initialisation values for structures and arrays.
See some examples:
- https://github.com/paparazzi/paparazzi/blob/master/conf/airframes/ENAC/quadrotor/cobraV2.xml#L94
- https://github.com/paparazzi/paparazzi/blob/master/conf/airframes/ENAC/quadrotor/cobraV2.xml#L171

You can also check my comment in this discussion: https://github.com/paparazzi/paparazzi/discussions/3189#discussioncomment-7685872
With this, you should be able to declare your walls directly in your airframe file.